package.json

```
@@ -20,8 +24,9 @@
     "lint:ts": "vale docs-src/typescript/*.md",
     "serve": "docusaurus serve",
     "start": "docusaurus start",
+    "snipsync": "snipsync",
     "swizzle": "docusaurus swizzle",
-    "worker": "pushd assembly && yarn worker && popd",
+    "worker": "cd assembly && yarn worker && cd ..",
```

Fixes cloud worker issue
```suggestion
    "worker": "cd assembly && yarn worker && cd ..",
    "cloud-worker": "cd assembly && ./worker.js --cloud && cd ..",
```